config/services.yaml

```diff
@@ -97,6 +97,9 @@ services:
     App\Service\Filesystem:
       public: true
 
+    App\Service\Index\Mesh:
```

Comment:
But now this isn't needed here anymore, should get auto discovered.

Reply:
ugh, sorry. 

handled by making it public in test env only.